programs/mango-v4/src/instructions/account_create.rs

```diff
@@ -29,7 +29,7 @@ pub fn account_create(
     account.fixed.delegate = Pubkey::default();
     account.fixed.set_being_liquidated(false);
 
-    account.expand_dynamic_content(token_count, serum3_count, perp_count, perp_oo_count)?;
+    account.expand_dynamic_content(token_count, serum3_count, perp_count, perp_oo_count, 0)?;
```

Comment:
maybe we should start with a default of 4?

Reply:
yeah, some default would be good. AccountCreateV2 I think, so it can be ui-controlled, like the others